plugins/pencilblue/controllers/actions/admin/content/media/delete_media.js

```diff
@@ -77,7 +77,7 @@ module.exports = function(pb) {
     };
 
     DeleteMediaController.prototype.removeLocal = function(media, mservice, cb) {
-        if (!media.is_file) {
+        if (!media.is_file && media.location.indexOf('http') !== 0 && media.location.indexOf('//') !== 0) {
```

Comment:
If you do this then how does the file actually get deleted out of the storage provider?

My guess is that there is no way for the providers to set a remote URL and a storage path.